src/index.html

```diff
@@ -7,12 +7,65 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/index.css"
     />
   </head>
   <body>
     <h1>Product cards</h1>
+
+    <main>
+      <div
+        data-qa="card"
+        class="card"
+      >
+        <img
+          src="images/imac.jpeg"
+          alt="Apple iMac 27-inch Retina 5K Monoblock"
+          class="card__img"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="stars-block">
+          <div class="stars">
+            <div class="star star--active"></div>
+            <div class="star star--active"></div>
+            <div class="star star--active"></div>
+            <div class="star star--active"></div>
+            <div class="star"></div>
```

Comment:
Fix classes and implement the modifier logic
```suggestion
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
```